modules/state_machine/states/state.gd

```diff
@@ -0,0 +1,17 @@
+extends Node
+
+func _ready():
+	set_process_input(false)
+	set_process(false)
+	set_physics_process(false)
```

Comment:
not sure if we should keep this, but it makes sense since the `kinematic_character.gd` delegates its calls to the states